### ISSUE TITLE
Exception: Invalid objective: nmo

### ISSUE THREAD

**Issue by sarvghotra** [aravindr93/RL-tasks#1]
https://github.com/aravindr93/RL-tasks/blob/master/actor-critic/ActorCritic.py#L67
This line has nmo loss; which is not available in keras so python raised error when I tried to run this.

**Comment by aravindr93** [comment]
You can add the loss to the objectives.py file. Details available in readme.md or actor-critic directory.

**Comment by sarvghotra** [comment]
I solved it by adding loss function in the ActorCritic.py file itself. I think this a cleaner way. Anyway this is a solved issue so I am closing this.
